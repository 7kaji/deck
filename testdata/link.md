# Link

- [sample for using image](https://github.com/k1LoW/deck/blob/main/testdata/images.md)
- [link_with_underscore](https://github.com/k1LoW/deck)
- [normal text **bold_text** more text](https://github.com/k1LoW/deck)
- [**bold_start** normal_middle **bold_end**](https://github.com/k1LoW/deck)

<!-- {"layout":"title-and-body"} -->
