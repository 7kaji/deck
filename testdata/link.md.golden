[
  {
    "layout": "title-and-body",
    "titles": [
      "Link"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "sample for using image",
                "link": "https://github.com/k1LoW/deck/blob/main/testdata/images.md"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "link_",
                "link": "https://github.com/k1LoW/deck"
              },
              {
                "value": "with_",
                "link": "https://github.com/k1LoW/deck"
              },
              {
                "value": "underscore",
                "link": "https://github.com/k1LoW/deck"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "normal text ",
                "link": "https://example.com"
              },
              {
                "value": "bold_",
                "bold": true,
                "link": "https://example.com"
              },
              {
                "value": "text",
                "bold": true,
                "link": "https://example.com"
              },
              {
                "value": " more text",
                "link": "https://example.com"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "bold_",
                "bold": true,
                "link": "https://example.com"
              },
              {
                "value": "start",
                "bold": true,
                "link": "https://example.com"
              },
              {
                "value": " normal_",
                "link": "https://example.com"
              },
              {
                "value": "middle ",
                "link": "https://example.com"
              },
              {
                "value": "bold_",
                "bold": true,
                "link": "https://example.com"
              },
              {
                "value": "end",
                "bold": true,
                "link": "https://example.com"
              }
            ],
            "bullet": "-"
          }
        ]
      }
    ],
    "headings": {
      "1": [
        "Link"
      ]
    }
  }
]